docs/superpowers/plans/2026-05-17-calibrator-review-time-features.md

```
@@ -0,0 +1,886 @@
+- Modify: `src/calibrate.rs:12,728-786` — remove import and `compute_thresholds` function
+
+- [ ] **Step 1: Delete threshold_config.rs**
+
+```bash
+rm src/threshold_config.rs
+```
+
+- [ ] **Step 2: Remove pub mod from lib.rs**
+
+In `src/lib.rs`, remove the line:
+```rust
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)